src/clusterfuzz/_internal/bot/fuzzers/centipede/engine.py

```diff
@@ -30,15 +30,15 @@
 _SERVER_COUNT = 1
 _RSS_LIMIT = 4096
 _ADDRESS_SPACE_LIMIT = 4096
-_TIMEOUT_PER_INPUT = 20
+_TIMEOUT_PER_INPUT = 60
 _DEFAULT_ARGUMENTS = [
     '--exit_on_crash=1',
     f'--fork_server={_SERVER_COUNT}',
     f'--rss_limit_mb={_RSS_LIMIT}',
     f'--address_space_limit_mb={_ADDRESS_SPACE_LIMIT}',
 ]
 
-_CRASH_REGEX = re.compile(r'[sS]aving input to:? [\n]?(.*)')
+CRASH_REGEX = re.compile(r'[sS]aving input to:?\s*[\n]?(.*)')
```

Comment:
As discussed, let's remove this `\n` bit since it's redundant.

Reply:
Did you end up making this change? Or doing this in a future PR?

Reply:
Just pushed it.
I thought I pushed it earlier, but I was apparently wrong.

Thanks for the reminder!